content/docs/command-reference/remote/modify.md

```diff
@@ -298,6 +298,35 @@ For more information on configuring Azure Storage connection strings, visit
 
 <details>
 
+### Click for Dropbox
+
+Please see [Setup a Dropbox DVC Remote](/doc/user-guide/setup-dropbox-remote)
+for a full guide on using Dropbox as DVC remote storage.
+
+- `url` - remote location, in the `dropbox://<dvc-subdirectory>` format:
+
+  ```dvc
+  $ dvc remote modify myremote url dropbox://my-project
+  ```
+
+- `chunk_size` - size of a single data chunk for the files to upload. Needs to
+  be a multiple of 4 MB, with a minimum of 4 MB and maximum of 150 MB. Default
+  `48`.
+
+See [Authorization](/doc/user-guide/setup-dropbox-remote#authorization) for more
+details.
+
+> If you synchronize your Dropbox account with your PC, it may be useful to
+> enable selective sync or smart sync. See
+> [Selective Sync](/doc/user-guide/setup-dropbox-remote#selective-sync) for more
+> details.
```

Comment:
Probably no need for this note since users can find it in the full guide and is not related to `remote modify`:

```suggestion
```

Reply:
I would move this suggestion to `remote add` then, to at least let users know about such setting.

Reply:
I don't think it's needed there either. Users can find this info in the full guide. They will probably already know if they're using selective/smart sync.